README.md

```diff
@@ -275,6 +280,176 @@ Reads if device geolocation is activated or not (read-only).
 console.log(pk.hasGeolocation); // true or false
 ```
 
+### `pk.patch.list()`
+
+List, filter and paginate patches.
+⚠️ Restricted to **private** API keys, **do not expose it to the browser**.
+
+```js
+// get all patches, paginated
+pk.patch.list().then((res) => {
+  console.log(res.results);
+});
+
+// filter and paginate patches
+pk.patch.list({
+  query: 'angeles',
+  countries: ['us'],
+  types: ['street'],
+  status: 'approved',
+  maxResults: 10,
+  offset: 0,
+}).then((res) => {
+  console.log(res.results);
+});
+```
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `query` | `string?` | Search query terms. |
+| `opts` | `key-value mapping` (optional) | Search options. |
+| `opts.status` | `('all' | 'pending' | 'approved')?` | Publication status. |
+| `opts.countries` | `string[]?` | Countries filter, array of [two-letter ISO](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country codes. |
+| `opts.types` | `string[]?` | Types filter, array of accepted values: `street`, `city`, `airport`, `bus`, `train`, `townhall`, `tourism`. Prepend `-` to omit a type like `['-bus']`. Unset to return all. |
+| `opts.maxResults` | `number?` | Maximum number of results to return. |
+| `opts.offset` | `number?` | Paginate results starting from the offset. |
+
+### `pk.patch.create()`
+
+Add a missing record or fix an existing one.
+⚠️ Restricted to **private** API keys, **do not expose it to the browser**.
+
+```js
+// Adding a missing record
+const record = {
+  type: 'street',
+  name: 'Example street',
+  city: 'Los Angeles',
+  county: 'Los Angeles',
+  administrative: 'California',
+  country: 'United States of America',
+  countrycode: 'us',
+  coordinates: '33.9955095,-118.472482',
+  zipcode: [90291],
+  population: 3849000, // optional
+};
+pk.patch.create(record, { status: 'approved' }).then((record) => {
+  console.log(record);
+});
+
+// Fixing an existing record
+pk.patch.create(
```

Comment:
I find the method signature a bit weird. Can accept a full record or a partial one, then the second argument is either an opts or a full record, and third only for fixes.

Reply:
What do you mean? Switching `origin` and `opts` arguments?

From this:

```js
patch.create(fullUpdate, opts);
patch.create(partialUpdate, origin, opts);
```

to this?

```js
patch.create(fullUpdate, opts);
patch.create(partialUpdate, opts, origin);
```

Reply:
I had put them in the 1st example order because devs will most likely provide an `origin` before `options` when creating a record, didn't want them to force the `patch.create(update, {}, origin)` which is ugly.

Reply:
Ok, ended-up swapping back the order in `patch.create` because my condition was failing.